services/robotchassis.md

```diff
@@ -0,0 +1,30 @@
+# Robot Chassis
+
+A bidirectional drive robot.
+
+	identifier: 0x19f4d06b
+	extends: _sensor
+
+## Registers
+
+    ro position @ reading {
+	    x: i16.16 cm
+		y: i16.16 cm
+    	vx: i16.16 cm/s
+        vy: i16.16 cm/s
+		heading: i16.16 °		
+    }
+    
+The current position and orientation of the robot.
+
+    rw enabled: bool @ intensity
```

Comment:
does it belong here? I think either we have a way to control position, or no control over motors at all.